pkg/sysregistriesv2/system_registries_v2_common.go

```diff
@@ -0,0 +1,12 @@
+//go:build !freebsd
```

Comment:
(Absolutely non-blocking: Naming this `pkg/sysregistriesv2/paths_{common,freebsd}.go` would work just as well, and be a tiny bit more descriptive.)